### PR TITLE
Add to Google Calendar

### PR DESCRIPTION
**Pull request by xuebingli**
@zendesk/volunteer-eng 

Addresses #59. 

* Adds a "Add to Google Calendar" link to event page. The link opens a pre-filled Google Calendar event form.
* Removes unused imports.

### CODE REVIEW

File: app/javascript/pages/Event/index.js [hunk 1]
```diff
@@ -148,8 +163,8 @@ const Event = ({ data: { loading, event, currentUser }, createSignup, destroySig
             </Section>
           </div>
           <div className={s.rightCol}>
-            <Section title="Information">
```
Comment (danielbreves) [inline app/javascript/pages/Event/index.js]: Is this never populated?

File: app/javascript/pages/Event/index.js [hunk 2]
```diff
@@ -148,8 +163,8 @@ const Event = ({ data: { loading, event, currentUser }, createSignup, destroySig
...
-              <div />
+            <Section title="Reminder">
```
Comment (danielbreves) [inline app/javascript/pages/Event/index.js]: Export? Integrations? 🤷‍♂️

**Review by danielbreves — APPROVED** [review]
Awesome!

File: app/javascript/pages/Event/index.js (hunk 1 above)
Comment (xuebingli) [reply app/javascript/pages/Event/index.js]: Yes, as far as I can see, thus I repurposed it for "Add to Google Calendar" button.

File: app/javascript/pages/Event/index.js [hunk 3]
```diff
@@ -36,6 +33,24 @@ const Section = ({ title, children }) => (
...
+  const templateUrl =
+    'http://www.google.com/calendar/event?action=TEMPLATE' +
+    `&text=${title}` +
+    `&dates=${formatDateTime(startsAt)}/${formatDateTime(endsAt)}` +
+    `&ctz=${timezone}` +
+    `&details=${description}` +
+    `&location=${location}`
+
+  return (
+    <a href={templateUrl} target="_blank" rel="nofollow">
```
Comment (nchieng) [inline app/javascript/pages/Event/index.js]: Might consider adding `noopener` and `noreferrer` rel attributes too.

File: app/javascript/pages/Event/index.js (hunk 2 above)
Comment (xuebingli) [reply app/javascript/pages/Event/index.js]: "Export" sounds good 👍

**Comment by xuebingli** [comment]
@nchieng 💇‍♂️

**Review by nchieng — APPROVED** [review]
👍

**Pull request merged**